crates/resolver/src/lib.rs

```diff
@@ -173,20 +173,7 @@
 //! complexities to figure out with IPv6. Once enabled, an mDNS `NameServer` will automatically be
 //! added to the `Resolver` and used for any lookups performed in the `.local.` zone.
 
-// LIBRARY WARNINGS
-#![warn(
-    clippy::default_trait_access,
-    clippy::dbg_macro,
-    clippy::print_stdout,
-    clippy::unimplemented,
-    clippy::use_self,
-    missing_docs,
```

Comment:
is it intentional to drop `missing_docs`? This isn't present in the workspace lints.

Reply:
Brought it back.